src/main/java/com/elmakers/mine/bukkit/action/builtin/AnimateAction.java

```diff
@@ -22,774 +20,4 @@ public SpellResult step(CastContext context) {
 	};
 
 	public static boolean DEBUG = false;
-
```

Comment:
Dead / commented out code isn't exactly useful either when we have git though.
I just removed the dead lines because of unused warnings and then the commented out code out of cleaning habit.